### PR TITLE
375 email link

### PR DESCRIPTION
**Pull request by lauraGgit**
## Summary
Resolves Issue #[375](https://github.com/18F/fs-open-forest-platform/issues/375)

*Describe the pull request here, including any supplemental information needed to understand it.*

## Impacted Areas of the Site
Email upon submission of a temp outfitter submission

## This pull request changes...
- [x] move when email gets sent so app status will be infact submitted
- [x] change urls for temp outfitter 
- [x] update tests

## This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [ ] The change has been documented
  - [ ] Associated OpenAPI documentation has been updated

### CODE REVIEW

File: server/src/email/templates/noncommercial/default-application-details.es6 [hunk 1]
```diff
@@ -7,7 +7,8 @@ module.exports = {
     return `
       Application details
       *********************************
-
+      
+      app status: ${application.status}
```
Comment (mgwalker) [inline server/src/email/templates/noncommercial/default-application-details.es6]: For content consistency, should this be `Application status: ${application.status}`?

Comment (lauraGgit) [reply server/src/email/templates/noncommercial/default-application-details.es6]: oh, this was a debugging statement- will remove!

Comment (lauraGgit) [reply server/src/email/templates/noncommercial/default-application-details.es6]: but actually it seems worthwhile to have ha!

Comment (lauraGgit) [reply server/src/email/templates/noncommercial/default-application-details.es6]: realizing this is covered by the conditional in the link text to the app

**Pull request merged**